x-pack/plugins/security_solution/public/management/components/endpoint_responder/lib/console_commands_definition.ts

```diff
@@ -77,6 +78,23 @@ const executeTimeoutValidator = (argData: ParsedArgData): true | string => {
   }
 };
 
+const filePathValidator = (argData: ParsedArgData): true | string => {
+  const emptyResult = emptyArgumentValidator(argData);
```

Comment:
you don't have to use `emptyArgumentValidator()` any more. just set `mustHave` property on your command definition and set it to `non-empty-string` and the console internals will take care of the validation.

Reply:
99e13666cb030e03427e48b3bf935b09b293207d